torchvision/transforms/transforms.py

```diff
@@ -1051,38 +1051,27 @@ def _check_input(self, value, name, center=1, bound=(0, float('inf')), clip_firs
         return value
 
     @staticmethod
-    @torch.jit.unused
-    def get_params(brightness, contrast, saturation, hue):
-        """Get a randomized transform to be applied on image.
+    def get_params(brightness: Optional[List[float]],
+                   contrast: Optional[List[float]],
+                   saturation: Optional[List[float]],
+                   hue: Optional[List[float]]
+                   ) -> Tuple[Tensor, Optional[float], Optional[float], Optional[float], Optional[float]]:
+        """Get the parameters for the randomized transform to be applied on image.
 
         Arguments are same as that of __init__.
```

Comment:
Nit: we may mention here that inputs here are not exactly as for `__init__` and require lists vs possible single floats for `__init__`

Reply:
Good catch, let me change this real quick.